src/features/click-to-load.js

```diff
@@ -80,8 +84,10 @@ class DuckWidget {
      *   The original tracking element to replace with a placeholder.
      * @param {string} entity
      *   The entity behind the tracking element (e.g. "Facebook, Inc.").
+     * @param {import('../utils').Platform} platform
+     *   The platform where Click to Load and the Duck Widget is running on.
```

Comment:
Nit: Maybe explain what a platform is in this context? E.g. "The browser platform, e.g. "android"."